gateway-service-health/src/main/java/org/apache/knox/gateway/service/health/PingResource.java

```diff
@@ -81,7 +87,26 @@ private Response getPingResponse() {
   }
 
   String getPingContent() {
-    return CONTENT;
+    return OK;
+  }
+
+  @GET
+  @Produces({TEXT_PLAIN})
+  @Path("gateway-status")
+  public Response status() {
+    response.setStatus(HttpServletResponse.SC_OK);
```

Comment:
The `response` field is always set to the same here and in `getPingResponse` too. You may want to refactor this out to an `init` method and call it only once (e.g. using the `PostConstruct` annotation as we do in other resource classes).

Reply:
Ignore this comment, please.